force-app/main/service/DonationHistoryService.cls

```diff
@@ -100,7 +100,26 @@ public with sharing class DonationHistoryService {
         } else {
             throw new UTIL_Permissions.InsufficientPermissionException(Label.commonInsufficientPermissions);
         }
-        
+    }
+
+    /** 
+    * @description method to return Donation History list.
+    * @param paymentListForOpportunity List<npe01__OppPayment__c> for current opportunity
+    * @return String with payment method for current opportunity
+    */
+    public String getRequiredPaymentMethodForOpportunities(List<npe01__OppPayment__c> paymentListForOpportunity){
+        String paymentMethodToReturn = '';
+		for (npe01__OppPayment__c p : paymentListForOpportunity) {
```

Comment:
We still have a indentation issue here. Configure VSCode to indent with 4 spaces instead of tabulation